R/commonly_used_db_data.R

```diff
@@ -0,0 +1,277 @@
+#################################
+### prepare coverage and fvps ###
+#################################
+
+##' Find latest touchstone given touchstone_name
+##'
+##' @title Determine touchstone
+##'
+##' @param con Database connection.  
+##' @param touchstone_name touchstone_name
+##' @export
+get_touchstone <- function(con, touchstone_name){
```

Comment:
I think we might have talked a bit about this before but generally how are you expecting these functions to be used? 

They are intended for the science team to use right. How are they creating a connection? Where will this functions be called from? It might be good to add some examples or a vignette here which talks about these in more detail.